cmd/bbolt/command_surgery_cobra.go

```diff
@@ -78,6 +81,48 @@ func surgeryRevertMetaPageFunc(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func newSurgeryCopyPageCommand() *cobra.Command {
+	copyPageCmd := &cobra.Command{
+		Use:   "copy-page <bbolt-file> [options]",
+		Short: "Copy page from the source page Id to the destination page Id",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("db file path not provided")
+			}
+			if len(args) > 1 {
+				return errors.New("too many arguments")
+			}
+			return nil
+		},
+		RunE: surgeryCopyPageFunc,
+	}
+
+	copyPageCmd.Flags().StringVar(&surgeryTargetDBFilePath, "output", "", "path to the target db file")
+	copyPageCmd.Flags().Uint64VarP(&surgerySourcePageId, "from-page", "", 0, "source page Id")
+	copyPageCmd.Flags().Uint64VarP(&surgeryDestinationPageId, "to-page", "", 0, "destination page Id")
+
+	return copyPageCmd
+}
+
+func surgeryCopyPageFunc(cmd *cobra.Command, args []string) error {
+	srcDBPath := args[0]
+
+	if surgerySourcePageId == surgeryDestinationPageId {
+		return fmt.Errorf("'--from-page' and '--to-page' have the same value: %d", surgerySourcePageId)
+	}
+
+	if err := common.CopyFile(srcDBPath, surgeryTargetDBFilePath); err != nil {
+		return fmt.Errorf("[copy-page] copy file failed: %w", err)
+	}
+
+	if err := surgeon.CopyPage(surgeryTargetDBFilePath, common.Pgid(surgerySourcePageId), common.Pgid(surgeryDestinationPageId)); err != nil {
+		return fmt.Errorf("copy-page command failed: %w", err)
+	}
```

Comment:
Shell we warn if the freelist is persisted and recommend dropping it ?

This might cause unreachable pages being reachable again.

Reply:
Good point! Added,
```
	fmt.Fprintf(os.Stdout, "WARNING: the free list might have changed.\n")
	fmt.Fprintf(os.Stdout, "Please consider executing `./bbolt surgery abandon-freelist ...`\n")
```

Reply:
Yes, let me do a minor refactoring in a followup PR,

https://github.com/etcd-io/bbolt/blob/88c6eb7fdcc3dc2c486c635adb4afd4fd8bf812a/cmd/bbolt/command_surgery_cobra.go#L218-L222